etcd.py

```diff
@@ -253,6 +255,8 @@ def etcd_arguments(self, data_dir, initial_cluster, cluster_state):
             self.peer_url,
             '-listen-client-urls',
             'http://0.0.0.0:{}'.format(self.client_port),
+            '-listen-metrics-urls',
```

Comment:
It breaks compatibility with old etcd, because the same script is used for different images containing different version.
Moreover, it even breaks upgrade 3.2 => 3.3.

Those arguments should be applied conditionally, i.e. only if etcd version is >= 3.3.
You can get version from environment: `ETCDVERSION`.